test/powershell/engine/SemanticVersion.Tests.ps1

```diff
@@ -178,10 +199,6 @@ Describe "SemanticVersion api tests" -Tags 'CI' {
             # Revision isn't supported
             { [SemanticVersion]::new([Version]::new(0, 0, 0, 4)) } | Should Throw # PSArgumentException
             { [SemanticVersion]::new([Version]::new("1.2.3.4")) } | Should Throw # PSArgumentException
-
-            # Build is required
```

Comment:
I don't know if that was an explicit decision or accidentally left out.  I don't see any reason why it shouldn't be supported as it seems 0.0.0 is a valid semver.  I'll add support since it's a small change even if not related to this change :)